### ISSUE TITLE
Display pre-loader before uploading

### ISSUE THREAD

**Issue by yddgd** [bytemedrive/android#144]
When selecting multiple files for upload (7 videos for about 1GB in total in this case), after confirmation, there is nothing happening for about 30 seconds, only after that, the loading is started. The "upload menu drawer" is displayed the whole time and the app looks frozen. 

Would be nice to have some indication of the fact that something is happening :) 

Maybe toaseter "Your files will be uploaded" just after confirming, not only after it it processed..
